lib/alephant/lookup/lookup_cache.rb

```diff
@@ -16,22 +16,25 @@ def initialize(config={})
         unless config_endpoint.nil?
           @client ||= ::Dalli::Client.new(config_endpoint, { :expires_in => ttl })
         else
-          logger.debug "Alephant::LookupCache::#initialize: No config endpoint, NullClient used"
```

Comment:
bit of weird notation here, it should probably be: `Alephant::LookupCache#initialize`